src/styles/mixins/_typography.scss

```diff
@@ -42,3 +42,30 @@
     line-height: $bdl-line-height;
     letter-spacing: .3px;
 }
+
+@mixin target-ie11 {
+    @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
+        @content;
+    }
+}
+
+@mixin line-clamp($max-lines: 2, $line-height: 1.5) {
```

Comment:
I was thinking something like `bdl-lineClamp` but maybe it'll be better to have a more descriptive name like `bdl-truncateMultiline` or `bdl-limitTextLines` or some other variation?